acceptance/setup/aio/pre-suite/010_Install.rb

```diff
@@ -1,38 +1,18 @@
 require 'puppet/acceptance/common_utils'
 require 'puppet/acceptance/install_utils'
 
+require 'beaker-puppet'
 extend Puppet::Acceptance::InstallUtils
+extend BeakerPuppet::Install::Puppet5
 
 test_name "Install Packages"
 
 step "Install puppet-agent..." do
-  opts = {
-    :puppet_collection    => 'PC1',
-    :puppet_agent_sha     => ENV['SHA'],
-    # SUITE_VERSION is necessary for Beaker to build a package download
-    # url which is built upon a `git describe` for a SHA.
-    # Beaker currently cannot find or calculate this value based on
-    # the SHA, and thus it must be passed at invocation time.
-    # The one exception is when SHA is a tag like `1.8.0` and
-    # SUITE_VERSION will be equivalent.
-    # RE-8333 may make this unnecessary in the future
-    :puppet_agent_version => ENV['SUITE_VERSION'] || ENV['SHA']
-  }
-  agents.each do |agent|
-    next if agent == master # Avoid SERVER-528
-    install_puppet_agent_dev_repo_on(agent, opts)
-  end
+  dev_builds_url  = ENV['DEV_BUILDS_URL'] || 'http://builds.delivery.puppetlabs.net'
+  sha = ENV['SHA']
+  install_from_build_data_url('puppet-agent', "#{dev_builds_url}/puppet-agent/#{sha}/artifacts/#{sha}.yaml", agents)
```

Comment:
I think we want `hosts` instead of `agents`. Previously, we explicitly installed `puppet-agent` dev repo on line 91, so it was installed as a dependency on `puppetserver`. But since SERVER-528 has been resolved, we should be able to install puppet-agent on all hosts, and install puppetserver after.

Reply:
I'm curious how this would work for ec2 hosts. It looks like we're already hitting builds.delivery.puppetlabs.net to install puppet-agent on ec2, but I was under the impression this wouldn't work...... I'm wondering if we should keep this as `agents`, and make sure to also install puppet-agent later on when we install puppet-server? This could also potentially cause ec2 to break as is....

Reply:
This wouldn't break the ec2 tests as is, since it's only operating on the agent platforms. Are the ec2 tests always limited to only the master platforms?

However, as is, I do need to add a line below to install puppet-agent on the server :(